CHANGES.rst

```diff
@@ -4,6 +4,18 @@ Changes
 0.12.1 (unreleased)
 ~~~~~~~~~~~~~~~~~~~
 
+- Add a script to check a database to see if a class is used as a
+  persistent object in it or not or if it is missing.
+- Various improvements on the previous script looking for broken
+  records and references.
+- Add script to check a database for broken records and cache the object
+  references information in a sqlite database file. This allows for
+  displaying forward and backward object references as wel as providing
+  an overview of currently broken object and by whom they are referred to.
+- Add support for a ZConfig configuration file to specify the
+  database. This bring support for RelStorage and other non-default
+  storage.
```

Comment:
This is a good idea, and I'd love to see it as a separate PR.

Language nitpickery: bring -> brings, storage -> storage formats.